ngrinder-controller/build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     compile (group: "org.apache.maven", name: "maven-embedder", version: "3.0.5")
     compile (group: "org.sonatype.aether", name: "aether-connector-asynchttpclient", version: "1.13.1")
     compile (group: "org.liquibase", name: "liquibase-core", version: "3.5.3")
+    compile (group: "org.hibernate", name: "hibernate-core", version: hibernate_version)
```

Comment:
Is this auto include with `spring-boot-starter-data-jpa`?

https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-data-jpa/2.1.8.RELEASE

Reply:
You are right. It includes in `spring-boot-starter-data-jpa`.
I'm gonna remove this dependency and bump up other spring related library versions.